src/api/entities/Asset/TransferRestrictions/Count.ts

```diff
@@ -53,7 +61,7 @@ export class Count extends TransferRestrictionBase<TransferRestrictionType.Count
    * Enables an investor count statistic for the Asset, which is required before creating restrictions
    *
    * The counter is only updated automatically with each transfer of tokens after the stat has been enabled.
-   * As such the initial value for the stat should be passed in
+   * As such the initial value for the stat should be passed in, which can be fetched with {@link Count.investorCount }
```

Comment:
If this gets merged after alpha -> beta, @link tag would require some changes to support wiki linking

Reply:
@polymath-eric could you fix this as @prashantasdeveloper mentions?